src/pages/release-notes.md

```diff
@@ -42,9 +48,9 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
 
 #### Compatible versions
 
-| Component                                    | Versions                                       |
-|----------------------------------------------|------------------------------------------------|
-| Eclipse Data Space Connector (Tractus-X EDC) | [0.7.3](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.3) [Recommended] <br /> [0.7.4](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.4) <br />   [0.7.5](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.7.5) [Latest] |
+| Component                                    | Versions                                                                                                                                                                                                                                                                      | Comments                                                                                                                     |
```

Comment:
I'd add a short explanation, on why we only list the EDC as a component - ("others will follow" oder sowas). Sonst ist man als "Laie" ggf. etwas verwirrt.

Reply:
I would have hoped that this line is enough  - should I rewrite it

> By default, the versions listed in the [Tractus-X changelog](https://eclipse-tractusx.github.io/CHANGELOG/#2408---2024-08-05) are the recommended versions. Non-compatible or other compatible versions of the Tractus-X release or the apps are listed below.